components/Pet.jsx

```diff
@@ -77,9 +76,8 @@ class Pet extends Component {
       pettype: 'frog',
       images: [],
       startDate: new Date("2024-03-16"), //Date the pet was created,  we need to get the info from pet creation
-      examDate: new Date("2024-05-18"), // Date the exam is due , we need to get the info from pet creation, please implement this
```

Comment:
I noticed that a lot of changes from main are being deleted. Make sure to be aware of what changes you want to merge.